doc/source/cookbook.rst

```diff
@@ -776,11 +776,14 @@ Resampling
 
 The :ref:`Resample <timeseries.resampling>` docs.
 
-`TimeGrouping of values grouped across time
-<http://stackoverflow.com/questions/15297053/how-can-i-divide-single-values-of-a-dataframe-by-monthly-averages>`__
```

Comment:
I would paybe try to update the stackoverflow post?

In any case, the Grouper docs in itself are not a good reference for "Using Grouper to group values across time"